Library/Homebrew/cmd/update-report.rb

```diff
@@ -238,19 +241,25 @@ def output_update_report
     return if new_repository_version.blank?
```

Comment:
```suggestion
    return if new_repository_version.blank?
    return if args.quiet?
```

Nothing happens below this except for printing messages, so let's just return here and remove all `--quiet` checks below